controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -666,10 +670,18 @@ bool wfa_ca::get_send_1905_1_tlv_hex_list(std::list<tlv_hex_t> &tlv_hex_list,
             // Type
             if (tlv_member_idx == 0) {
                 tlv_hex.type = &it->second;
+                if (!validate_hex_notation(*tlv_hex.type, 1)) {
```

Comment:
What will happen if the testbed will send `tlv_type,0x02` or `0x0002`? Is it invalid? Why do we want to make things harder for the end user?

Reply:
Yes, it is invalid because the UCC should send the hex notation string with the number of octets it wants to be written to the buffer.
for two bytes: 0x0002,
for one byte: 0x02

Reply:
> Yes, it is invalid because the UCC should send the hex notation string with the number of octets it wants to be written to the buffer.

Neither the CAPI specification nor the EasyMesh test plan really say that for type and value the number of hex characters must exactly correspond to the size of these two. However, all the examples have it with the exact size, and UCC does generate it with the exact size, and it makes our implementation a bit simpler if we can rely on the exact size. So, I agree with Moran that it's good like this. If it turns out that UCC does generate something else, we can adapt.

BTW this check was already there before. It's just a refactoring.